src/shims/unix/linux/eventfd.rs

```diff
@@ -63,15 +64,15 @@ impl FileDescription for Event {
                 throw_unsup_format!("eventfd: blocking is unsupported");
             }
         } else {
-            // Prevent false alarm in data race detection when doing synchronisation via eventfd.
+            // Future `read` calls will synchronize with this write, so update the FD clock.
```

Comment:
This is a "read", I think you mixed up the 2 comments?
```suggestion
            // Synchronize with past `write` calls.
```

Reply:
Ah yes, thanks for catching this. It is likely that my eyes parse the word ``read`` and decided to put it in ``read`` function ><